WMFModel/WMFArticle+CoreDataClass.h

```diff
@@ -0,0 +1,12 @@
+#import <Foundation/Foundation.h>
+#import <CoreData/CoreData.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface WMFArticle : NSManagedObject
```

Comment:
Yeah - this is what "Create NSManagedObjectSubclass" does now... not sure if we can revert to the old behavior.